### PR TITLE
Fix(UI) settings tooltip

### LINKED ISSUE #5931: Change background of page settings button dropdown
Page settings dropdown at the right end looks like a tooltip, make it a bit lighter in dark mode.

image

Component in question is `kestra/ui/src/components/filter/components/Settings.vue`.
New filters are (for now) only used on `Executions` page.

### PR DESCRIPTION
**Pull request by satvik2131**
Added bs gray 300
closes https://github.com/kestra-io/kestra/issues/5931

### CODE REVIEW

File: ui/src/components/filter/components/Settings.vue [hunk 1]
```diff
@@ -36,6 +36,7 @@
 <style lang="scss">
 .settings-dropdown {
     width: 200px;
+    background-color: var(--bs-grey-300) !important;
```
Comment (Piyush-r-bhaskar) [inline ui/src/components/filter/components/Settings.vue]:
```suggestion
    background-color: var(--bs-gray-300) !important;
```

**Review by Piyush-r-bhaskar — CHANGES_REQUESTED** [review]

File: ui/src/components/filter/components/Settings.vue (hunk 1 above)
Comment (Piyush-r-bhaskar) [reply ui/src/components/filter/components/Settings.vue]: **Need to check the bg color of switch in off for light theme.**
image

border need to checked with gray-300.
image

Maybe arrow should also be in gray-300
image

Comment (satvik2131) [reply ui/src/components/filter/components/Settings.vue]: yes , firstly in the previous conversation , it was decided to first change the color and then we will plan how it is going to be in final .

**Comment by MilosPaunovic** [comment]
Hi @satvik2131

I'll have to close this PR, as I've merged the https://github.com/kestra-io/kestra/issues/6260 this morning, so this is now conflicting with your changes.

Thanks for all the efforts! 🚀

**Comment by satvik2131** [comment]
> Hi @satvik2131
> 
> I'll have to close this PR, as I've merged the https://github.com/kestra-io/kestra/issues/6260 this morning, so this is now conflicting with your changes.
> 
> Thanks for all the efforts! 🚀 

Oh okay , no problem .

**Pull request closed without merging**